app/gui/src/project-view/components/GraphEditor.vue

```diff
@@ -738,6 +738,11 @@ const contextMenuActions: ActionName[] = [
   & .vertical {
     flex: auto;
     overflow-x: hidden;
+    /*
+     * If this isn't set then during the BottomPanel opening/closing transition the native scrollbar
+     * may be displayed momentarily, causing the viewport to "shake".
+     */
+    overflow-y: hidden;
```

Comment:
This comment speaks about issue during transition, but the rules is always declared, not only during active animations. If this is not an issue, you could just use `overflow: clip` without x/y axis split.